### ISSUE TITLE
Drug object is missing drug identifiers. 

### ISSUE THREAD

**Issue by MatthiasWiesmann** [schemaorg/schemaorg#3327]
The `Drug` object is missing national drug identifiers. 

* For US drugs, this would be a NDC (National Drug Code).
* For Germany PZN (Pharmazentralnummer)

**Comment by RichardWallis** [comment]
I note from [this listing](http://apps.gs1.org/GDD/Pages/clDetails.aspx?semanticURN=urn:gs1:gdd:cl:AdditionalTradeItemIdentificationTypeCode) that there a few more than two of these drug code identifier schemes.
It would not scale to have individual properties for all of them.

We already have an `identifier` property and a way to assign values to it using the [PropertyValue](https://schema.org/PropertyValue) type.  Why not use that?

For example:
```
{
 "@context": "https://schema.org/",
 "@type": "Drug",
 "name": "Morphine Sulfate",
 "identifier": {
 "@type": "PropertyValue",
   "propertyID": "NDC",
   "value":  "76329-1911"
  }
}
```

**Comment by MatthiasWiesmann** [comment]
This goes back to the discussion about gtin and similar identifiers. At the very least, there should be better examples.

**Comment by RichardWallis** [comment]
> This goes back to the discussion about gtin and similar identifiers. 

Agree, the whole area of [identifier](https://schema.org/identifier) and domain specific subtypes is one that did not evolve easily and had already raised some how would it scale issues, when the `PropertyValue` approach was introduced.

> At the very least, there should be better examples.

Totally agree, would you be up for providing some convincing `Drug` related examples?

**Comment by MatthiasWiesmann** [comment]
Sure.

**Comment by MatthiasWiesmann** [comment]
I added pull request https://github.com/schemaorg/schemaorg/pull/3333 with improved documentation for Drug.

**Comment by RichardWallis** [comment]
Implemented in PR #3333
